### PR TITLE
Changes: [t1434] Fixed firefox newscaster logo CSS, converted to better html5

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/c51ca865bfe4340/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/c51ca865bfe4340/output.txt

Total script time: 0.57 mins

Server is running on http://pr445.butyr.org/templates/test.html

File: templates/supported/newscaster/css/style.css [hunk 1]
```diff
@@ -234,16 +232,12 @@ section[role="suplementary"] p:hover {
...
-  -ms-transform: rotateX(90deg) translateZ(50px);
-  transform: rotateX(90deg) translateZ(50px);
+.cube > div > div {
+  display: table;
+  width: 100%;
+  height: 100%;
 }
-
-.cube > div:nth-child(2) {
+.cube > div[data-panel="side"]:nth-child(1) {
```
Comment (secretrobotron) [inline templates/supported/newscaster/css/style.css]: Why don't you just use classes instead of "data-panel"?

Comment (secretrobotron) [inline templates/supported/newscaster/css/style.css]: I did the same thing here, which works in FF and Chrome: http://secretrobotron.github.com/crunch/

Comment (k88hudson) [inline templates/supported/newscaster/css/style.css]: This is just a marker, not really important for anything. I will take it out.

File: templates/supported/newscaster/index.html [hunk 2]
```diff
@@ -32,13 +32,29 @@
...
       <section class="cube">
-        <div></div>
-        <div><h2>knn news</h2></div>
-        <div><h2>knn news</h2></div>
-        <div><h2>knn news</h2></div>
-        <div><h2>knn news</h2></div>
-        <div></div>
-        <div></div>
+        <div>
+          <div calls="cube-side">
```
Comment (secretrobotron) [inline templates/supported/newscaster/index.html]: `calls` should probably be `class`. Do these even need a class here anymore?

File: templates/supported/newscaster/index.html [hunk 3]
```diff
@@ -32,13 +32,29 @@
...
+            <h2>knn news</h2>
+          </div>
+        </div>
+        <div calls="cube-side">
```
Comment (secretrobotron) [inline templates/supported/newscaster/index.html]: Here too.

File: templates/supported/newscaster/index.html (hunk 2 above)
Comment (k88hudson) [inline templates/supported/newscaster/index.html]: ohhh man.

Comment (k88hudson) [inline templates/supported/newscaster/index.html]: Good to keep it though, yes. There are too many nested divs and things might get confusing otherwise

**Pull request merged**